docs/client-sdk/javascript/tutorials/label-conversations.md

```diff
@@ -0,0 +1,55 @@
+
+Apps that assign conversation IDs in this way have an effect on apps that use a portable inbox model. For example, a user might see multiple separate and ongoing conversations they are having with the same address, but originating in different apps.
+
+To help clarify this user experience, you can display conversation labels based on conversation IDs and metadata.
+
+To illustrate this scenario, let's take a look at a couple of conversations between `amal.eth` and `bola.eth`.
+
+`amal.eth` and `bola.eth` can have an ongoing conversation in App ABC. Let's say that App ABC sets a `conversationId` of `abc.dev/dm` and uses it to filter conversations to display only conversations its users created using App ABC.
+
+![Mockup of App ABC setting a conversation ID of abc.dev/dm and filtering conversations by it. The app displays amal.eth's inbox with a "Hey..." conversation from bola.eth](img/app-abc.png)
+
+`amal.eth` and `bola.eth` can also have a separate and ongoing conversation in App XYZ. Let's say that App XYZ sets a `conversationId` of `xyz.dev/dm` and uses it to filter conversations to display only conversations its users created using App XYZ.
+
+![Mockup of App XYZ setting a conversation ID of xyz.dev/dm and filtering conversations by it. The app displays amal.eth's inbox with a "Yes..." conversation from bola.eth](img/app-xyz.png)
+
+Let's then take a look at App 123, which doesn't filter conversations and displays all conversations for a user regardless of where the conversations were created. In this case, the inbox for `amal.eth` might look something like this:
+
+![Mockup of App 123 with no conversation ID or filtering. The app displays amal.eth's inbox with two conversations with bola.eth: One "Hey..." and one "Yes..."](img/app-123.png)
+
+This scenario may display a confusing user experience for `amal.eth`, surfacing two separate conversations they are having with the same address, `bola.eth`.
+
+To help distinguish the multiple conversations `amal.eth` is having with the same address, you can implement labels, based on conversation IDs and metadata, that display for each conversation.
+
+![Mockup of App 123 with no conversation ID or filtering, but with conversation ID and metadata-based labels implemented. The app displays amal.eth's inbox with two conversations with bola.eth: One labeled as ABC: "Hey..." and one labeled as XYZ: "Yes..."](img/app-123-convo-labels.png)
+
+These labels can help `amal.eth` recognize the source of each conversation with `bola.eth` and understand why multiple conversations are displaying.
+
+**Use labels to differentiate multiple conversations between a pair of addresses:**
+
+For example:
+
+<!--do we have a code sample we can include here? perhaps from this XMTP Chat repo PR https://github.com/xmtp/example-chat-react/pull/140? Would love to be able to publish this tutorial for the XMTP V2 Dec. 5 due date - possible?-->
```

Comment:
@bhavya2611 Might you have a code sample you can share for this doc example about displaying a conversation label, please? =)

Do you think it would be possible to provide it in time for a Dec. 5 publication date? If not, I can hold this tutorial back until a code sample is available. Thank you!

Reply:
From Bhavya <3

```
// To get the metadata
const conversationId = conversation?.context?.conversationId

// example conversationId = lens.dev/dm/0x123-0x456

const domain = conversationId.split("/")[0]

// domain would be lens.dev in the above case which is the label 
```